qpylib/json_qpylib.py

```diff
@@ -54,7 +53,6 @@ def _extract_type_from_context(context):
     return None
 
 def get_jsonld_type(jsonld_type):
-    global JSONLD_TYPES
```

Comment:
Just a question about this - will this still work if it's not set up as a global variable? What if for example you were using it in Flask/GUnicorn with multiple threads? Not entirely sure how this works but have hit issues with Python globals before.